cpp/test/prims/silhouette_score.cu

```diff
@@ -183,25 +195,32 @@ class silhouetteScoreTest
   LabelT *d_labels = nullptr;
   int nRows;
   int nCols;
+  int nElements;
   double truthSilhouetteScore = 0;
   double computedSilhouetteScore = 0;
+  double batchedSilhouetteScore = 0;
   cudaStream_t stream;
+  raft::handle_t handle;
+  int chunk;
+  std::shared_ptr<MLCommon::deviceAllocator> allocator;
 };
 
 //setting test parameter values
 const std::vector<silhouetteScoreParam> inputs = {
-  {4, 2, 3, raft::distance::DistanceType::EucExpandedL2, 0.00001},
-  {4, 2, 2, raft::distance::DistanceType::EucUnexpandedL2Sqrt, 0.00001},
-  {8, 8, 3, raft::distance::DistanceType::EucUnexpandedL2, 0.00001},
-  {11, 2, 5, raft::distance::DistanceType::EucExpandedL2, 0.00001},
-  {40, 2, 8, raft::distance::DistanceType::EucExpandedL2, 0.00001},
-  {12, 7, 3, raft::distance::DistanceType::EucExpandedCosine, 0.00001},
-  {7, 5, 5, raft::distance::DistanceType::EucUnexpandedL1, 0.00001}};
+  {4, 2, 3, raft::distance::DistanceType::EucExpandedL2, 4, 0.00001},
```

Comment:
These distances were recently renamed and the PR was merged into 0.18. Should be a simple rename (`Euc` was removed and the `Expanded`/`Unexpanded` pushed after the distance name. Also `Unexpanded` was removed from `L1`)